### PR TITLE
Titles print on each line

### PR DESCRIPTION
**Pull request by breyana**

### CODE REVIEW

File: movie-search.js [hunk 1]
```diff
@@ -4,15 +4,29 @@ const cheerio = require('cheerio')
...
 
+const printTitles = (rawData) => {
+  const $ = cheerio.load(rawData)
+  const titles = []
+  //There are multiple sections called .findSection, we only need the movies
+  //so we look for the first .findSection, and grab the .result_text from it
+  $('.findSection')
+    .first()
+    .find('.result_text')
+    .each((i, elem) => {
```
Comment (pllearns) [inline movie-search.js:15]: I used `map` here, so I am curious as to the difference, I did read the docs, but I still would like to compare the output with you sometime.

**Review by pllearns — COMMENTED** [review]
Looks great. I did make one comment about the use of `.each` which seems fine, but I had used `.map` on mine, so more of a curiosity of resulting output.

File: movie-search.js (hunk 1 above)
Comment (diop) [reply movie-search.js:15]: @deadlyicon pointed out that you would have problems parsing certain queries with ```+ search +```

Comment (jaredatron) [reply movie-search.js:15]:
```js
const URL = `http://www.imdb.com/find?ref_=nv_sr_fn&q=${encodeURIComponent(search)}&s=all`
```

Comment (jaredatron) [reply movie-search.js:15]: query is probably a better variable name than search

**Pull request merged**